src/content/en/2020/fonts.md

```diff
@@ -22,3 +22,266 @@ queries: 04_Fonts
 #featured_stat_label_3: TODO
 unedited: true
 ---
+## Introduction
+
+Text is at the heart of most web sites, and typography is the art of presenting that text in a way that's visually appealing and effective. Creating good typography requires choosing the appropriate fonts. Web fonts give designers a tremendous range of fonts to choose from. As with all resources, there are performance and compatibility concerns, but done right, the benefit is well worth it. In this chapter, we'll dive into data to show how web fonts are being used, and in particular how they're optimized.
+
+Web font technology has been fairly mature, with incremental improvements in compression and other technical improvements, but new features are arriving. Browser support for variable fonts has become quite good, and this is the feature that's seen the most growth in the previous year.
+
+## Where are web fonts being used?
+
+Web font usage has been growing steadily over time, with 82% of web pages for desktop using web fonts, and mobile at 80%.
+
+{{ figure_markup(
+  image="fonts-web-fonts-usage.png",
+  caption="Web font usage over time.",
+  description="Scatter plot showing fraction of mobile and desktop web pages containing web fonts, as a function of time.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=1465601804&format=interactive",
+  sheets_gid="655527383",
+  sql_file="04_01.web_fonts_usage.sql"
+  )
+}}
+
+Usage of web fonts is fairly consistent around the world, with a few outliers. The charts below are based on the median number of kilobytes of web fonts per web page, which can be an indicator of lots of fonts, large fonts, or both.
+
+https://discuss.httparchive.org/t/how-does-web-font-usage-vary-by-country/1649
+
+{{ figure_markup(
+  image="fonts-web-fonts-usage-by-country.png",
+  caption="Web fonts usage by country.",
+  description="A map of the world showing the amount of web font usage for each country, measured as median kilobytes of web font data.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=1005874204&format=interactive",
+  sheets_gid="655527383",
+  sql_file="04_02.web_fonts_usage_by_country.sql"
+  )
+}}
+
+The single top country is South Korea, which is not all that surprising given their consistently high internet speeds and the fact that Korean (Hangul) fonts are almost an order of magnitude larger than Latin. As we'll see in the "popular typefaces" chapter, Korean fonts are two of the top 14 most popular, further evidence they really like web fonts. Web font usage in Japan and Chinese-speaking countries is considerably lower, likely because CJK fonts are even larger and connections are not as fast.
```

Comment:
High speeds and low latency (the latter being more important as speeds drop).

Reply:
"because CJK fonts are even larger" can't really say that Chinese, Japanese and Korean fonts are even larger than Korean :) 

Suggest instead "because Chinese and Japanese fonts are vastly larger (the median font size being 10,000 times larger than the median Latin size).

Reply:
Suggest adding "This means web font usage in Japan is very low, and usage in China is effectively zero. Although [recent developments](https://www.w3.org/TR/2020/NOTE-PFE-evaluation-20201015/) may make web fonts usable in both countries, within a couple of years."

Reply:
10,000 times larger? I don't see how that can be. I'm going to put "over 100" in as a placeholder for now, but can refine it based on data.

Reply:
@raphlinus I think 1,000x is safely on the mark (at least): I've seen many CJK web fonts in the 20MB+ range for a single instance. Might be reasonable to say 'on average several hundred times larger' to emphasize the scale of the difference.

Reply:
How do we feel about "the median font size can be 1000 times or more larger than the median Latin size"?

Reply:
> 10,000 times larger? I don't see how that can be. I'm going to put "over 100" in as a placeholder for now, but can refine it based on data.

Good catch, the figure was indeed based on data but I was calculating a percentage. 1,834,496 * 100 / 17,660 = 10,388 %

"The TrueType or OpenType fonts used for alphabetic languages had a median size of 17,660 bytes, with a spread from 15,784 bytes (5th percentile) to 184,152 bytes (95th percentile). The median size once WOFF2 compressed was 8,360 bytes"

"The TrueType or OpenType fonts used for CJK languages had a median size of 1,834,496 bytes, with a spread from 16,832 bytes (5th percentile) to 5,524,2080 bytes (95th percentile). The median size once WOFF2 compressed was 768,492 bytes."

https://www.w3.org/TR/2020/NOTE-PFE-evaluation-20201015/#font-langtype